prime-router/src/test/kotlin/azure/BlobAccessTests.kt

```diff
@@ -173,13 +191,12 @@ class BlobAccessTests {
         every { anyConstructed<BlobClientBuilder>().buildClient() } returns mockedBlobClient
 
         val result = BlobAccess.uploadBlob(testName, testBytes, testContainer, testEnv)
+        // upload a second blob to the same container to test container client reuse
```

Comment:
It'd be nice if this test verified that `create` is only called once. As it stands, it may cover those lines, but the test isn't actually verifying the client reuse.

Reply:
Good call, added that verification.